twitchio/user.py

```diff
@@ -693,6 +693,42 @@ async def create_prediction(
         )
         return Prediction(self._http, data[0])
 
+    async def fetch_schedule(
+        self,
+        segment_ids: List[str] = None,
+        start_time: str = None,
+        utc_offset: str = None,
```

Comment:
this should take as an int

Reply:
I'll change it to a int on input and convert to string on http request